__tests__/duration.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env jest */
+
+const fs = require('fs');
+const path = require('path');
+const _ = require('lodash');
+
+const durationJSON = require('../dist/duration.json');
+
+const durationFiles = [
+  'duration.map.scss',
+  'duration.custom-properties.css',
+  'duration.scss',
+  'index.d.ts',
+];
+
+durationFiles.map((filename) =>
+  it(`renders ${filename} correctly`, () => {
+    const contents = fs.readFileSync(
+      path.join(__dirname, '..', 'dist', filename),
+      {
+        encoding: 'utf-8',
+      },
+    );
+    expect(contents).toMatchSnapshot();
+  }),
+);
+
+describe('JSON object representation', () => {
+  it('renders similar JSON objects', () => {
+    expect(
+      _(durationJSON)
+        .toPairs()
+        .sortBy(0)
+        .fromPairs()
+        .value(),
+    ).toMatchSnapshot();
+  });
+
+  it('outputs integers in JSON', () => {
```

Comment:
Not sure how useful this test actually is... would probably be better to loop through each key in the object and assert that each is a `number`. Couldn't find any `jest` methods for this - wasn't sure of the best way to author that assertion.

Reply:
Would an asserrstion on Object.values + every work?

Reply:
Brilliant! This is what I did:

```
  it('outputs integers in JSON', () => {
    const everyValueIsInteger = Object.values(durationJSON).every((value) =>
      Number.isInteger(value),
    );
    expect(everyValueIsInteger).toBe(true);
  });
```